modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/util/ConfigInitializerContextListener.java

```diff
@@ -70,7 +73,7 @@ public void contextInitialized(ServletContextEvent servletContextEvent) {
 
         ServletContext servletContext = servletContextEvent.getServletContext();
         File propertyFile = new File(getClass().getClassLoader().getResource(
```

Comment:
Use some meaningful name in place of "propertyFile" to name this particular variable.